### DISCUSSION TITLE
Road to 1.0.0 - Invitation to collaborate

### DISCUSSION THREAD

**Discussion by tomasvotava** [tomasvotava/fastapi-sso#122] [Announcements]
Hey everyone,

Since the first commit, I've learned a lot and now recognize and am quite ashamed of some early decisions in the codebase that I believe we can improve. This package has become an integral part of my own projects, therefore I want to address the things that stink about it 👊.

I'm thrilled to see the community not only using `fastapi-sso` but also contributing to its improvement through excellent PRs. Your involvement is truly valued and always welcome ❤️ .

With that in mind, I think it's the right time to break things apart and build something better. We're at a point where making some backward-incompatible changes could lead to a more stable, reliable, and refined `fastapi-sso`.

@parfeniukink, your significant refactor in PR #120 has caught my attention. How about we take this opportunity to make `fastapi-sso` backward-incompatible and pursue an even more comprehensive refactor? Would you be interested in collaborating on this, helping with the design, and later, the coding? 💪 

I extend this invitation to anyone in the community who's interested. Your input is invaluable, and I'd be glad to have more collaborators on board for this important phase. 👐 

Looking forward to your thoughts and contributions. 🚀

**Comment by parfeniukink** [comment]
@tomasvotava 

Hey bro.
I am really happy that you are so cool on that 💪
Making this stable and maintainable is a good choice for any open source which is not going to die in a while 😄
